components/fieldgroup/stories/fieldgroup.test.js

```diff
@@ -7,47 +6,51 @@ export const FieldGroupSet = Variants({
 	testData: [
 		{
 			testHeading: "Default",
+			inputType: "radio",
 		},
 		{
-			testHeading: "Horizontal",
+			testHeading: "Vertical checkboxes",
+			layout: "vertical",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
+		},
+		{
+			testHeading: "Horizontal radios",
+			layout: "horizontal",
+			inputType: "radio",
+		},
+		{
+			testHeading: "Horizontal checkboxes",
 			layout: "horizontal",
-			items: [
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "apple",
-					label: "Apples are best",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "banana",
-					label: "Bananas forever",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "cherry",
-					label: "Cherries ftw",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-			],
-		},
-		{
-			testHeading: "Label position: side",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
+		},
+		{
+			testHeading: "Radios label position: side",
 			label: "Pick one:",
 			labelPosition: "side",
-			helpText: "Select an option to continue.",
+			inputType: "radio",
+		},
+		{
+			testHeading: "Checkboxes label position: side",
+			label: "Choose:",
+			labelPosition: "side",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
```

Comment:
```suggestion
{
	testHeading: "Checkboxes label position: side",
	labelPosition: "side",
	inputType: "checkbox",
},
```

We can probably remove `label` too and use the default `label` arg.